reactioncommerce/reaction
# Feat 4719 client ui payment methods

Resolves #4719   
Impact: **minor**  
Type: **feature|refactor**

## Issue
Client UI does not use the new payment methods API.

## Solution
Updated client UI to use the new payment method endpoints.

## Breaking changes
The names of the payment methods are slightly different.

## Testing
1. Look at the payment method list on checkout
2. Look at the default payment methods field in the admin panel
3. Toggle payment methods in the admin panel and verify their state changes
4. Make sure the discount codes section still works as it used to, as this is handled separately and just shown at the same level as the other payment methods
